docs/tutorials/index.md

```diff
@@ -91,6 +91,7 @@ Select API:&nbsp;
    * [Image similiarity search with InfoGAN](/tutorials/gluon/info_gan.html)
 * Practitioner Guides
     * [Gotchas using NumPy](/tutorials/gluon/gotchas_numpy_in_mxnet.html)
+    * [Performance Tips & Tricks](/tutorials/gluon/performance.html)
```

Comment:
Better to align with the name in the page "Gluon Performance Tips & Tricks"

Reply:
Updated, thanks.

Reply:
Although we're in the Gluon section so adds a bit of noise.